javascript/node/selenium-webdriver/phantomjs.js

```diff
@@ -15,6 +15,28 @@
 // specific language governing permissions and limitations
 // under the License.
 
+/**
+ * @fileoverview Defines a {@linkplain Driver WebDriver} client for the PhantomJS
+ * web browser. By default, it is expected that PhantomJS exists in your working directory.
+ * 
+ *  __Using a Custom PhantomJS Binary__
+ * 
+ * If you have PhantomJS.exe placed somewhere other than the root of your working directory,
+ * you can build a custom Capability and attach the executable's location to the Capability
+ * 
+ * For example, if you're using the phantomjs-prebuilt module (https://www.npmjs.com/package/phantomjs-prebuilt) from npm:
```

Comment:
For example, if you're using the [phantomjs-prebuilt](https://www.npmjs.com/package/phantomjs-prebuilt)
    module from npm: